applications/DEMApplication/custom_elements/spheric_particle.h

```diff
@@ -131,7 +131,7 @@ void TransformNeighbourCoorsToClosestInPeriodicDomain(const ProcessInfo& r_proce
 virtual bool CalculateRelativePositionsOrSkipContact(ParticleDataBuffer & data_buffer);
 
 using DiscreteElement::Initialize; //To avoid Clang Warning. We tell the compiler that we are aware of the existence of this function, but we overload it still.
```

Comment:
@maceligueta I guess this can be removed afterwards

Reply:
sure! Should I?

Reply:
I think it is better to do it in a future PR, this one is already large and contains delicate changes